package.json

```diff
@@ -19,7 +19,9 @@
     "test:ci": "node --expose-gc --trace-warnings --unhandled-rejections=strict ./node_modules/jest/bin/jest.js --bail --ci --colors --expand --errorOnDeprecated --coverage --reporters=default --maxWorkers=1 --forceExit --logHeapUsage",
     "test:manual": "jest --expand --colors --watch",
     "test:debug": "node --inspect-brk ./node_modules/jest/bin/jest.js --expand --colors --watch --runInBand",
-    "prepublishOnly": "node scripts/npm-package.js"
+    "prepublishOnly": "node scripts/npm-package.js",
+    "deploy": "gh-pages -d dist",
+    "predeploy": "npm run build:prod"
```

Comment:
let's have the only one script `deploy` it will do all combined magic.

Reply:
Removed predeploy script